### ISSUE TITLE
When selecting a date it selects the previous day instead of the selected data :(

### ISSUE THREAD

**Issue by SomenteRafael** [Faltenreich/Diaguard#49]
When selecting a date it selects the previous day instead of the selected date* :(

Vídeo:
https://github.com/Faltenreich/Diaguard/assets/43616612/551d24e1-364c-4e12-b3fc-f3a040d49f69

**Comment by Faltenreich** [comment]
I agree that this should not happen. May I ask which time zone you are using?

**Comment by SomenteRafael** [comment]
UTC-3

**Comment by Faltenreich** [comment]
@SomenteRafael I was able to fix this issue in the entry form's date picker and the export form's date range picker. A bugfix will arrive with version 3.13.1 in the upcoming hours. Thank you very much for reporting this issue. All the best!
